ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main.yaml

```
@@ -7,7 +7,7 @@ releases:
   latest:
     integration:
       include_built_images: true
-      name: "4.17"
+      name: "4.18"
```

can't we call this latest  and set all the jobs to latest

and next time just update the version 
i.e 
```suggestion
      name: "latest"
```
job example
```
- as: e2e-metal-assisted-cnv-latest
  optional: true
  steps:
```

---

I don't think there is an image stream named "latest" -
```
grep -C 3 -r "integration:" ./ | grep "name:" | grep latest
./ci-operator/config/quay/quay-bridge-operator/quay-quay-bridge-operator-master__ocp-latest.yaml-      name: "4.13"
./ci-operator/config/quay/quay-operator/quay-quay-operator-master__ocp-latest.yaml-      name: "4.13"
./ci-operator/config/quay/quay-operator/quay-quay-operator-redhat-3.8__ocp-latest.yaml-      name: "4.13"
./ci-operator/config/quay/quay-operator/quay-quay-operator-redhat-3.9__ocp-latest.yaml-      name: "4.13"
```

---

it must be x.y version

---

https://steps.ci.openshift.org/ci-operator-reference#line1266